src/builders_v2/linux_vm/kernel.rs

```diff
@@ -82,16 +95,15 @@ impl Kernel {
     }
 
     /// Whether kernel was precompiled or not.
+    #[allow(unused)]
     pub fn is_precompiled(&self) -> bool {
         matches!(self, Self::Precompiled { .. })
     }
 
     // TODO: use this function.
     // TODO: maybe use libgit instead of executable?
```

Comment:
Are these TODO comments still relevant?

Reply:
The first one is not (I will remove it), but the second one is relevant.